src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,16 +97,26 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        return LocalDate.now().equals(someDate) ? someDate + " is today"
```

Comment:
create a variable in this method not to call LocalDate.now multiple times

Reply:
done